addons/binding/org.openhab.binding.honeywellwifithermostat/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="honeywellwifithermostat"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="honeythermo">
+        <label>Honeywell WiFi Thermostat</label>
+        <description>Control your Honeywell WiFi thermostat over the internet with OpenHAB.</description>
```

Comment:
This should be a description of the Thing itself, e.g. what device or device type is it exactly (e.g. a certain model from Honeywell or ALL there wifi thermostats (like Lyric?)).